### ISSUE TITLE
plot_spectrum doesn't always work with png option

### ISSUE THREAD

**Issue by aburgasser** [pyspextool/pyspextool#68]
In making the QA pages, I noticed that individual spectral plots created in the extraction stage will not get produced if the qa_extension = .png. This isn't an issue for the combined spectra qa plots though

**Comment by aburgasser** [comment]
to clarify, files are created but they don't contain any useable information (can't be viewed)

**Comment by michaelcushing** [comment]
For context, the reason it was only creating pdf files is because there are times with multi-aperture files are created (think extended source with multiple apertures across a galaxy) and I wanted the plots of all the apertures to be in a single file so I forced it to be pdf.  But obviously that is too restrictive so I have updated to allow it to do both multipage pdf or dump single .whatever files to disk with the suffix _apertureXX.whatever.  If there is only one aperture, i.e. the vast majority of point sources, it won't add the _apertureXX part.  I don't know whether this will cause problems downstream when it comes to automating things but we can revisit this if the problem arises.
